doc/living_in_belgium.md

```diff
@@ -0,0 +1,71 @@
+**If you are coming in the school period,** and you have the student document for transportation (certificate of provisional registration), you can buy an abonnement between two stations (e.g. LLN-Brussels zone). Be aware that it is not the same ticket as the summer period, it is only valid between stations that you choose. 
+### BUS (TEC)
+There is a direct bus from Etterbeek station and Roodebeek Station. 
+
+> [!TIP]
+> If you are confused with which ticket to buy, be aware that the bus ticket for going between Brussels and LLN is the express one, which is €5. 
+
+#### BRUSSELS - LLN by bus and train
+The easiest neighbourhoods in Brussels to reach LLN by train are in the south-east part of the city (Ixelles/Etterbeek/Schuman). 
+There is also a bus (E11) going from Delta (neighborhood Auderghem/watermael-Boitsfort) to LLN in 35min
+Or a bus (E12) going from Woluwé (Roodebeek/Stockel/Alma) to LLN in 30min. 
+
+### Renting car/ van
+- If you need a van ( with a driver + helper) to move your stuff in and around Brussels/Belgium or also to other european countries: https://www.thebulletin.be/rent-man-van-you-hate-moving-we-love-it-removals-simon-19
+      ["I find this service to be extremely professional & helpful" - a satisfied customer /lab member]
+
+- Olivier is also renting his car on Caramigo 
+
+- In case some of you are interested in a good deal to rent cars flexibly: this is an option I recently learned about: https://www.cambio.be/en-bxl
+
+## Accommodation
+The first choice is where do you want to settle in? Most typical options are Brussels, Louvain la Neuve, Wavre, Namur, etc... Most of the people in the group live in Bruxelles. Talk to the lab members about this to have their opinions!
+If you are looking to rent your own place, try http://www.immoweb.be. You can filter by location, price, as well as "furnished" or "unfurnished". Note that if you take an unfurnished flat it's common to have absolutely NOTHING in it. No blinds, no light fixtures, potentially no kitchen! Note that kitchens are often described as “semi-equipped” or “hyper-equipped”, this refers to what type of appliances are there (dishwasher, fridge, etc).
+Some other websites to check if you are looking for a room/apartment where to live either in Brussels or in Louvain-la-Neuve:
+* https://www.facebook.com/groups/385369448291405/?fref=ts
+* https://www.facebook.com/groups/virtueljojo/
+* https://www.facebook.com/groups/195527370457521/
+* https://www.facebook.com/groups/188519461195986/
+* https://www.facebook.com/groups/134093889938198
+* https://www.facebook.com/groups/1798484627017398
+* http://www.appartager.be
+* http://www.expatarrivals.com/belgium/brussels/areas-and-suburbs-in-brussels
+
+
+### Temporary accommodation
+First of all, check the UCLouvain webpage for accommodation: https://uclouvain.be/en/study/logement-0.html You can see lots of options there. 

```

Comment:
added this line and link because the previous link of the UCLouvain accommodation webpage wasn't working

Reply:
Check that these links work in the rendered version